contracts/src/v0.8/ccip/pools/GHO/UpgradeableBurnMintTokenPool.sol

```diff
@@ -44,28 +43,24 @@ contract UpgradeableBurnMintTokenPool is Initializable, UpgradeableBurnMintToken
     if (i_allowlistEnabled) _applyAllowListUpdates(new address[](0), allowlist);
   }
 
-  /// @notice This function allows the owner to mint `amount` tokens on behalf of the pool
-  /// and transfer them to `to`. This is GHO-Specific and is called to match the facilitator
-  /// level of the new pool with the old pool such that it can burn the bridged supply once
-  /// the old pool is deprecated. The old pool is then expected to burn `amount` of tokens
-  /// so that it can be removed as a facilitator on GHO (ideally using `directBurn`).
-  /// @dev This is only called while offboarding an old token pool (or facilitator) in favor
-  /// of this pool.
+  /// @notice Mint an amount of tokens with no additional logic.
+  /// @dev This GHO-specific functionality is designed for migrating bucket levels between
+  /// facilitators. The new pool is expected to mint amount of tokens, while the old pool
+  /// burns an equivalent amount. This ensures the facilitator can be offboarded, as all
+  /// liquidity minted by it must be fully burned.
+  /// @param to The address to which the minted tokens will be transferred.
   /// @param to The address to which the minted tokens will be transferred. This needs to
-  /// be the old token pool,
-  /// or the facilitator being offboarded.
+  /// be the old token pool, or the facilitator being offboarded.
   /// @param amount The amount of tokens to mint and transfer to old pool.
   function directMint(address to, uint256 amount) external onlyOwner {
     IBurnMintERC20(address(i_token)).mint(to, amount);
   }
 
-  /// @notice This function allows the owner to burn `amount` of the pool's token. This is
-  /// expected to be called while migrating facilitators by offboarding this facilitator in
-  /// favor of a new token pool.
-  /// @dev New token pool should mint and transfer liquidity to this pool (since this pool
-  /// does not hold tokens at any point in time) which can be burnt and hence will reduce
-  /// the facilitator bucket level on GHO. The naming convention is inspired from  that in
-  /// LockRelease type token pools for the sake of consistency.
+  /// @notice Burn an amount of tokens with no additional logic.
```

Comment:
can we add this docs to #21 so they are completely aligned?

Reply:
[ca73ec8](https://github.com/aave/ccip/pull/21/commits/ca73ec8c4f7dc0f6a99ae1ea0acde43776c7b9bb)